parts/plugins/x-apache.py

```diff
@@ -247,7 +247,6 @@ def _build_third_party_modules(self):
 
             configure_command = [
                 './configure', '--prefix=' + self.installdir,
-                '--with-apxs2={}/bin/apxs'.format(self.installdir),
```

Comment:
Yeah you should probably not cannibalize the apache plugin here-- FPM doesn't depend on apache, so it doesn't belong here.